bin/node-sass

```diff
@@ -229,63 +229,41 @@ function getOptions(args, options) {
  */
 
 function watch(options, emitter) {
-  var buildGraph = function(options) {
-    var graph;
-    var graphOptions = {
-      loadPaths: options.includePath,
-      extensions: ['scss', 'sass', 'css']
-    };
-
-    if (options.directory) {
-      graph = grapher.parseDir(options.directory, graphOptions);
-    } else {
-      graph = grapher.parseFile(options.src, graphOptions);
-    }
+  var handler = function(files) {
+    files.added.forEach(function(file) {
+      var watch = gaze.watched();
+      if (watch.indexOf(file) === -1) {
+        gaze.add(file);
+      }
+    });
 
-    return graph;
+    files.changed.forEach(function(file) {
+      if (path.basename(file)[0] !== '_') {
```

Comment:
We could move this logic into `lib/watcher` but if we were to extract it into it's own package I can see uses cases for people wanting to poke changed partials i.e. `url` rewriting, or some kind webpack magic.